shuup/core/fields/__init__.py

```diff
@@ -25,6 +25,8 @@
 
 IdentifierValidator = RegexValidator("[a-z][a-z_]+")
 
+DECIMAL_FIELD_DECIMAL_PLACES = 9
```

Comment:
Also this could be `SHUUP_DECIMAL_FIELD_PLACES` since this has nothing to do with Django I suppose?